### PR TITLE
WINDUPRULE-815: new test app and groovy spring-web rule for shaded jars

### PR DESCRIPTION
**Pull request by m-brophy**
https://issues.redhat.com/browse/WINDUPRULE-815

### CODE REVIEW

**Comment by PhilipCattanach** [comment]
@m-brophy Is there a new test ruleset for the new groovy rule?

**Comment by PhilipCattanach** [comment]
Ignore my comment! My mistake

File: rules-reviewed/quarkus/springboot/springboot-web-to-quarkus.windup.groovy [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import org.jboss.windup.reporting.category.IssueCategory
+import org.jboss.windup.reporting.category.IssueCategoryRegistry
+import org.jboss.windup.reporting.config.Hint
+import org.jboss.windup.reporting.config.Link
+import org.jboss.windup.rules.apps.java.condition.SourceMode
+import org.ocpsoft.rewrite.context.EvaluationContext
+
+final IssueCategory mandatoryIssueCategory = new IssueCategoryRegistry().getByID(IssueCategoryRegistry.MANDATORY)
+
+ruleSet("springboot-web-to-quarkus-groovy")
```
Comment (jmle) [inline rules-reviewed/quarkus/springboot/springboot-web-to-quarkus.windup.groovy:22]: @m-brophy maybe some spacing between code blocks would make the rule easier to read

**Comment by m-brophy** [comment]
retest

**Comment by PhilipCattanach** [comment]
@m-brophy - Great stuff Mark. I have reviewed and tested the changes and all is well. I will merge them to master as soon as the automated checks complete successfully. There may be a short delay while we get the JDK11 build infrastructure updated.

**Comment by m-brophy** [comment]
retest

**Comment by m-brophy** [comment]
retest

**Comment by m-brophy** [comment]
retest

**Comment by m-brophy** [comment]
retest

**Comment by m-brophy** [comment]
retest

**Pull request merged**